cmd/rotatecerts/operations.go

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+package rotatecerts
+
+import (
+	"fmt"
+	"math/rand"
+	"time"
+
+	"github.com/pkg/errors"
+	appsv1 "k8s.io/api/apps/v1"
+	v1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// RotateServiceAccountTokens deletes service account tokens referenced by kube-system daemonsets/deployments
+// and triggers a rollout of daemonsets/deployments that mount service account tokens.
+//
+// Service account tokens are signed by the cluster CA, deleting them after the CA is rotated ensure that
+// KCM will regenerate tokens signed by the new CA.
+func RotateServiceAccountTokens(client Client, namespaces []string) error {
```

Comment:
Because the `Client` type is an interface, thoughts on unit testing these funcs?

Reply:
correct, I am just waiting on collecting feedback before I spend time on UTs